test/celeritas/optical/Scintillation.test.cc

```diff
@@ -300,15 +300,16 @@ TEST_F(ScintillationTest, basic)
     auto const& data = params->host_ref();
     EXPECT_FALSE(data.scintillation_by_particle());
 
+    auto const pre_step = this->build_pre_step();
+
     // Pre-generate optical distribution data
     ScintillationPreGenerator generate(
         this->make_particle_track_view(post_energy_, pdg::electron()),
         this->make_sim_track_view(step_length_),
         post_pos_,
-        opt_matid_,
         edep_,
         data,
-        this->build_pre_step());
+        pre_step);
```

Comment:
Good catch by the way—I realized this is why one of my tests was spitting out `NaN`...